### PR TITLE
add better software post

### PR DESCRIPTION
**Pull request by andrea-p**

### CODE REVIEW

File: posts/2018-02-23-serverless-path-building-better-software.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Ideally, we’d be able to adopt a tool that has a strong open-source community and/or commercial company behind it—but is easily, and drastically, customizable.
+
+This answer might seem obvious, but we just aren’t there. The growing proliferation of open APIs did some of this groundwork, but it’s still far from easy to customize most tooling.
+
+Serverless, though, has the potential to make this ideal state a reality.
+
+If your source management, CI/CD, and project management system all exposed events in a uniform fashion, which you could easily react to with a serverless function deployed to any platform of your choice—well, I’d say that’s pretty ideal.
+
+We have a ways to go to achieve this, but projects like [Auth0’s Extend], [CNCF’s CloudEvents], and many more are making progress towards this goal.
```
Comment (andrea-p) [inline posts/2018-02-23-serverless-path-building-better-software.md]: @worldsoup Can you add the links here?

File: posts/2018-02-23-serverless-path-building-better-software.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Serverless architectures today are essentially microservice architectures: groups of functions that have common functionality. They are deployed together, and share infrastructure resources such as an API gateway.
+
+I believe that as serverless tooling evolves, these services will become much smaller, much more specific, and will cover a much wider set of use cases (e.g. a function that deploys and configures a specific piece of infrastructure, or performs a load-test). As these services become smaller and more specific, they also become more easily shared, configured, and consumed by people other than the original developer.
+
+This will eventually take us to a future in which micro-services can be easily shared, deployed, and reused—both publicly, and privately within an organization.
+
+This would dramatically reduce the amount of time we spend re-inventing the wheel and allow us to focus more on unique functionality and business value.

+
+There are currently efforts along these lines underway, such as [Standard Lib]. It’ll be really exciting to see where these go in the future.
```
Comment (andrea-p) [inline posts/2018-02-23-serverless-path-building-better-software.md]: @worldsoup Can you add link here too?

**Comment by andrea-p** [comment]
schedule(02-23-2018 6:00am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **02-23-2018 at 6:00amundefined (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/529#issuecomment-367999700)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**